cmd/spire-agent/cli/api/fetch_jwt.go

```diff
@@ -80,3 +77,29 @@ func (c *fetchJWTCommand) fetchJWTBundles(ctx context.Context, client *workloadC
 	}
 	return stream.Recv()
 }
+
+func printPrettyResult(results ...interface{}) error {
+	errMsg := "internal error: cli printer; please report this bug"
+
+	svidResp, ok := results[0].(*workload.JWTSVIDResponse)
+	if !ok {
+		fmt.Println(errMsg)
+		return errors.New(errMsg)
+	}
+
+	bundlesResp, ok := results[1].(*workload.JWTBundlesResponse)
+	if !ok {
+		fmt.Println(errMsg)
+		return errors.New(errMsg)
+	}
+
+	for _, svid := range svidResp.Svids {
+		fmt.Printf("token(%s):\n\t%s\n", svid.SpiffeId, svid.Svid)
```

Comment:
Errors from all of these Printf's need to be returned in order to detect a broken pipe

Reply:
I'm happy to do it, but I think it constitutes a change in behavior? We did not fail on this previously

Reply:
Oh right, yes. Sorry, I just remembered this code is part of the custom pretty printer.